Digitall.Persistance.EF/Abstractions/IDataSeedService.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Digitall.Persistance.EF.Abstractions
+{
+    public interface IDataSeedService
```

Comment:
I think the seed interface should be in the Application assembly. 
The implementation in Persistence as it is now. 
The idea of clean approach is separation of concerns - we put interfaces of Application layer and implementation in Infrastructure. You can use Abstractions assembly when you create GenericRepository and similar stuff

Reply:
Good point, you're correct. I forgot I'm using it from the presentation layer as well...